### PR TITLE
Refactor datastore

### PR DESCRIPTION
**Pull request by bdestombe**

### CODE REVIEW

File: src/dtscalibration/datastore.py [hunk 1]
```diff
@@ -213,6 +213,12 @@ def sections(self):
     def sections(self):
         self.attrs["_sections"] = yaml.dump(None)
 
+    @sections.setter
+    def sections(self, value):
+        msg = "Not possible anymore. Instead, pass the sections as an argument to \n" \
+              "ds.dts.calibrate_single_ended() or ds.dts.calibrate_double_ended()."
```
Comment (BSchilperoort) [inline src/dtscalibration/datastore.py:219]: Nice, `calibrate_` is a much better name :)

**Review by BSchilperoort — APPROVED** [review]
Nice work, just some relatively small comments from me. 👍

**Comment by bdestombe** [comment]
Do we need to merge this one before your IO split PR? #203

**Comment by BSchilperoort** [comment]
> Do we need to merge this one before your IO split PR? #203

Yup. That would be easiest.

**Pull request merged**